src/View/Association/index.html.twig

```diff
@@ -0,0 +1,48 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+
+{# Présentation #}
+
+
+    <h1 class="titleAssociation">L'ASSOCIATION EN BREF...</h1>
+    <div class="container-fluid">
+        <div class="row d-flex justify-content-around">
+            <div class="imagePresentation">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/69/Country-Dancing-Arthur_Murray_Dance_Studio_in_The_Woodlands_TX1080x720.jpg/800px-Country-Dancing-Arthur_Murray_Dance_Studio_in_The_Woodlands_TX1080x720.jpg" class="card-img-top" alt="...">
+            </div>
+            <div class="jumbotron jumbotron-fluid bg-white">
+                <div class="container">
+                    <h2 class="titlePresentation">En quelques mots...</h2>
+                    <p class="lead">Country Club Route 45 est un club de danse country et  de line dance de la région orléanaise, une association née en 2007
+                        du désir de partager la passion de danser dans le respect des traditions et également avec la joie que ce hobby peut apporter
+                        à un groupe de danseurs qui ont les mêmes goûts. L'association a toujours aujourd'hui un but simple : notre idéal est la promotion
+                        de la danse country et de la line dance, et le partage de notre passion. Le défunt Robert Wanstreet, fondateur de la danse country en France a accepté
+                        d'être le parrain du club dès sa création, chose qu'il n'avait jamais faite auparavant.</p>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+{# Animatrices #}
+
+
+{# Bureau #}
+
+    <h3 class="titleMember">Voici le bureau</h3>
+    <div class="container-fluid">
+        <div class="row d-flex justify-content-around">
+            {% for associations in associations %}
```

Comment:
non association in associations ! et le nommer member(s) partout au lieu de association(s) serait plus logique vu que c'est un MemberManager